packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -43,8 +43,6 @@ export const DrawerPanelContent: React.SFC<DrawerPanelContentProps> = ({
           className
         )}
         hidden={!isExpanded}
```

Comment:
Hey @boaz0 looks good, the only issue I see is that `hidden` is applied to the panel by default unless the drawer `isExpanded`. Which means it's also being applied when the drawer is `isStatic`. With a static drawer, the panel is always expanded, so you end up with this:

<img width="1768" alt="Screen Shot 2020-04-24 at 12 27 44 PM" src="https://user-images.githubusercontent.com/35148959/80240386-5f5d0680-8627-11ea-8c87-4b0491a35371.png">

Reply:
Got it, I will fix that. :)